internal/impl/lang/bloblang.go

```diff
@@ -1,12 +1,17 @@
 package lang
 
 import (
+	"crypto/rand"
 	"fmt"
+	"io"
 	"strings"
+	"time"
 
 	"github.com/bwmarrin/snowflake"
 	"github.com/bxcodec/faker/v3"
 	"github.com/gosimple/slug"
+	"github.com/oklog/ulid"
+	frand "golang.org/x/exp/rand"
```

Comment:
`math/rand.Source` is not safe for use from multiple goroutine. The alternative is to use this package which implements a [LockedSource](https://pkg.go.dev/golang.org/x/exp/rand#LockedSource)